### DISCUSSION TITLE
Strimzi Kafka Certificate renewal

### DISCUSSION THREAD

**Discussion by vv00639813** [strimzi/strimzi-kafka-operator#6021] [Q&A]
We are planning to renew certificate for Cluster CA and Client CA with below commands, 

kubectl annotate secret KAFKA-CLUTSER-NAME-cluster-ca-cert strimzi.io/force-renew=true
kubectl annotate secret KAFKA-CLUSTER-NAME-clients-ca-cert strimzi.io/force-renew=true

we also need to renew below secrets as we have external clients connect to Kafka, so please let me know the procedure to renew below secrets, also let me know if we need to update any other secret.

kafka-broker-secrets
kafka-public-creds

**Comment by scholzj** [comment]
I'm not sure what are these secrets - they are not created by Strimzi as part of the Kafka cluster. So it is hard to say what should you do with them.
